source/operations/cert-manager/cert-manager-tenants.rst

```diff
@@ -0,0 +1,249 @@
+   
+   Replace the string ``tenant-1`` throughout the procedures to reflect the name of your tenant.
+
+Prerequisites
+-------------
+
+- `kustomize <https://kustomize.io/>`__ installed
+- ``kubectl`` access to your ``k8s`` cluster
+- Completed the steps to :ref:`set up cert-manager <minio-setup-certmanager>`
+- The MinIO Operator installed and :ref:`set up for cert-manager <minio-certmanager-operator>`.
+
+1) Create the tenant namespace CA Issuer
+----------------------------------------
+
+Before deploying a new tenant, create a Certificate Authority and Issuer for the tenant's namespace.
+
+1. If necessary, create the tenant's namespace.
+
+   .. code-block:: shell
+      :class: copyable
+
+      kubectl create ns tenant-1
+
+2. Request a Certificate for a new Certificate Authority with ``spec.isCA`` set to ``true``.
+
+   Create a file called ``tenant-1-ca-certificate.yaml`` with the following contents:
+
+   .. code-block:: yaml
+      :class: copyable
+      :emphasize-lines: 7,8
+
+      # tenant-1-ca-certificate.yaml
+      apiVersion: cert-manager.io/v1
+      kind: Certificate
+      metadata:
+        name: tenant-1-ca-certificate
+        namespace: tenant-1
+      spec:
+        isCA: true
+        commonName: tenant-1-ca
+        secretName: tenant-1-ca-tls
+        duration: 70128h # 8y
+        privateKey:
+          algorithm: ECDSA
+          size: 256
+        issuerRef:
+          name: selfsigned-root
+          kind: ClusterIssuer
+          group: cert-manager.io
+
+3. Apply the file to the cluster.
+
+   .. code-block:: shell
+      :class: copyable
+
+      kubectl apply -f tenant-1-ca-certificate.yaml
+
+2) Create the ``Issuer``
+------------------------
+
+The ``Issuer`` issues the certificates within the tenant namespace.
+
+1. Generate a resource definition for an ``Issuer``.
+
+   Create a file called ``tenant-1-ca-issuer.yaml`` with the following contents:
+
+   .. code-block:: yaml
+      :class: copyable
+
+      # tenant-1-ca-issuer.yaml
+      apiVersion: cert-manager.io/v1
+      kind: Issuer
+      metadata:
+        name: tenant-1-ca-issuer
+        namespace: tenant-1
+      spec:
+        ca:
+          secretName: tenant-1-ca-tls
+
+2. Apply the ``Issuer`` resource definition to the cluster.
+
+   .. code-block:: shell
+      :class: copyable
+
+      kubectl apply -f tenant-1-ca-issuer.yaml
+
+3) Create a certificate for the tenant
+--------------------------------------
+
+Request that cert-manager issue a new TLS server certificate for MinIO.
+The certificate must be valid for the following DNS domains:
+
+- ``minio.<namespace>``
+- ``minio.<namespace>.svc``
+- ``minio.<namespace>.svc.<cluster domain>``
+- ``*.<tenant-name>-hl.<namespace>.svc.<cluster domain>``
+- ``*.<namespace>.svc.<cluster domain>``
+- ``*.<tenant-name>.minio.<namespace>.svc.<cluster domain>'``
+
+.. important::
+
+   Replace the filler strings with values for your tenant: 
```

Comment:
Do we have a usual term for nonspecific placeholder text? 

Or perhaps something like "Replace the placeholder values as described below"

Reply:
Trying an alternative. I think "placeholder text" is probably the clearest expression.